charts/comet-site-v1/values.yaml

```diff
@@ -32,6 +32,23 @@ resources:
     cpu: 50m
     memory: 200Mi
 
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+  # Configuration for the container that will mount the service account token
+  volumeMounts:
+    # Name of the volume mount for the service account token
+    name: "service-account"
+    # Path where the service account token will be mounted inside the container
+    mountPath: "/var/run/secrets/kubernetes.io/serviceaccount"
+    # Ensures the mounted service account token is read-only
+    readOnly: true
```

Comment:
Which of these values would a user (of the comet charts) want to change?

I suppose none, so we should not give the user these options.

Reply:
f57790d765615a2e056db25c009a20fabf4dd7a8